src/Pickles/Pickles/FeatureParser.cs

```diff
@@ -90,5 +92,18 @@ private string DetermineLanguage()
             }
             return language;
         }
+
+        private Feature RemoveFeatureWithIgnoreTag(Feature result)
+        {
+            if (result.Tags.Any(t => t == $"@{configuration.IgnoreTag}"))
+                return null;
+
+            var wantedFeatures = result.FeatureElements.Where(fe => fe.Tags.All(t => t != $"@{configuration.IgnoreTag}")).ToList();
```

Comment:
This makes the tag exclusion case sensitive. Do we want that? I'm not sure ... what do you think?